brayns/io/MorphologyLoader.cpp

```diff
@@ -455,7 +455,7 @@ class MorphologyLoader::Impl
      * not apply
      * @param compartmentReport Compartment report to map to the morphology
      * @param scene Scene to which the morphology should be loaded into
-     * @return True if the loading was successfull, false otherwise
+     * @return True if the loading was successful, false otherwise
```

Comment:
[successful](http://www.thesaurus.com/browse/successful) is the valid form of the word. Also check [successfull-successful-is-this-a-uk-us-difference](https://english.stackexchange.com/a/4619).

Reply:
@rolandjitsu I'm a little confused. The 2nd link says:
> I would say successfull is a typo. Neither Merriam-Webster nor Wiktionary mention it as an alternative spelling. The British National Corpus has 10695 cites for successful and exactly one for successfull. According to OneLook, 33 dictionaries have an entry for successful, but only Wordnik has a few cites for successfull (without a definition).

Reply:
If you check the first two links mentioned in the answer, none have a definition for `successfull`.  And the entry `successful` is mentioned in 33+ dictionaries. I guess that means the the proper form of this word should be `successful`.

But the answer notes if we use the adverb, then we use two `l` characters: `successfully`.

Reply:
exactly, so my revision here is valid.

Reply:
Haha, yes. Sorry, I got confused for a second there, I saw `successful` -> `successfull` not the other way around.